lib/ecto/adapters/sql.ex

```diff
@@ -245,6 +245,63 @@ defmodule Ecto.Adapters.SQL do
     end
   end
 
+  @doc """
+  Executes an EXPLAIN statement for the given query according to its kind and the
+  adapter in the given repository.
+
+  ## Examples
+
+      iex> Ecto.Adapters.SQL.explain(:all, repo, Post)
```

Comment:
```suggestion
      iex> Ecto.Adapters.SQL.explain(:all, Repo, Post)
```

Reply:
I'll review that doc to add examples for myxql and tds, but thanks for the suggestion :)